### ISSUE TITLE
AttributeError: 'urbackup_server' object has no attribute 'get_response'

### ISSUE THREAD

**Issue by Dasc3er** [uroni/urbackup-server-python-web-api-wrapper#8]
Calling the *_download_file* method after login, the following error appears:
```
AttributeError: 'urbackup_server' object has no attribute 'get_response'
```

The cause is the following line, where the method *get_response* is called instead of *_get_response*
https://github.com/uroni/urbackup-server-python-web-api-wrapper/blob/42a287d23a1153df636c193695615d99b7c75e4d/urbackup_api/__init__.py#L104

**Comment by Dasc3er** [comment]
Futhermore, the file downloaded by *download_installer* thourgh *_download_file* is in binary format.

**Comment by YWatchman** [comment]
The function isn't defined, must've been a typo. Has been fixed in my PR.

**Comment by uroni** [comment]
Fixed with fe8f7c53005da421bd7a1882993e31498b62a385
